lib/Doctrine/ORM/Tools/Console/Command/ValidateSchemaCommand.php

```diff
@@ -76,6 +77,19 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $ui->text('<comment>[SKIPPED] The database was not checked for synchronicity.</comment>');
         } elseif (! $validator->schemaInSyncWithMetadata()) {
             $ui->error('The database schema is not in sync with the current mapping file.');
+
+            if ($output->getVerbosity() >= OutputInterface::VERBOSITY_VERBOSE) {
+                $updates = $validator->getUpdateSchemaList();
+
+                $rows = [];
+                foreach ($updates as $idx => $sql) {
+                    $rows[] = [(int) $idx + 1, $sql];
```

Comment:
In what case is `$idx` not an integer, and what will happen if it isn't? For instance if it's `'a'`, `'b'`, `'c'`, what will happen?

Reply:
Hello gregoire, I was thinking, the update command uses: 

            foreach ($sqls as $sql) {
                $ui->text(sprintf('    %s;', $sql));
            }

So, to be consistent we should do the same here. Moreover, why not using $ui->listing($sqls)? Which has almost the same output and avoid doing a loop.

Reply:
Maybe it did not exist when that piece of code was written? Having a consistent output would be great, yeah.

Reply:
Done.